models/articles.models.js

```diff
@@ -29,3 +29,14 @@ exports.selectArticleByID = (id) => {
             return article;
         });
 };
+
+exports.selectArticleComments = (id) => {
+    return db
+        .query(
+            `SELECT comments.comment_id, comments.votes, comments.created_at, comments.author, comments.body, comments.article_id FROM comments JOIN articles ON comments.article_id = articles.article_id WHERE articles.article_id = $1 ORDER BY comments.created_at DESC;`,
+            [id]
+        )
+        .then(({ rows }) => {
+            return rows;
```

Comment:
This should be a left join. This is because if there are any articles without comments then it wouldn't be included as it wouldn't have anything to join onto.